docs/comparison.md

```diff
@@ -0,0 +1,141 @@
+you'd have to lean on community libraries or maintaining/implementing them yourself.
+
+### Framework Bindings
+
+| ---                           | urql                             | Apollo             | Relay             |
+| ----------------------------- | -------------------------------- | ------------------ | ----------------- |
+| React Bindings                | ✅                               | ✅                 | ✅                |
+| React Hooks Support           | ✅                               | ✅                 | ✅ (experimental) |
+| React Suspense (Experimental) | ✅ (experimental on client-side) | 🛑                 | ✅                |
+| Preact Support                | ✅                               | 🔶                 | 🔶                |
+| Svelte Bindings               | ✅                               | 🟡 `svelte-apollo` | 🛑                |
+| Vue Bindings                  | 🛑 (planned)                     | 🟡 `vue-apollo`    | 🟡 `vue-relay`    |
+| Initial Data on mount         | ✅                               | ✅                 | ✅                |
+
+Interestingly all three libraries heavily support React as they were all started from the React
+community outwards, but Apollo and Vue benefit from community bindings for different frameworks a
+lot. `urql`'s community is still growing and while that means it won't see large projects as
+Apollo's `vue-apollo` in the short term, we're aiming for first-party support for these bindings.
+
+This is a common criticism that we hear about `urql`, and it's true that our community isn't as
+extensive, but we attempt to lean on good and well funded maintenance at Formidable, active
+investment, and good documentation. So you may have to evaluate whether you can get any questions
+answered quickly enough on [GitHub Discussions](https://github.com/FormidableLabs/urql/discussions)
+for instance.
+
+### Caching and State
+
+| ---                                                     | urql                                                                  | Apollo             | Relay                                          |
+| ------------------------------------------------------- | --------------------------------------------------------------------- | ------------------ | ---------------------------------------------- |
+| Caching Strategy                                        | Document Caching, Normalized Caching with `@urql/exchange-graphcache` | Normalized Caching | Normalized Caching (schema restrictions apply) |
+| Added Bundle Size                                       | +5.9kB (with Graphcache)                                              | +0 (default)       | +0 (default)                                   |
+| Automatic Garbage Collection                            | ✅                                                                    | ✅                 | ✅                                             |
```

Comment:
Is it automatic in apollo, thought you had to call .gc

Reply:
That's true, I just checked and it _isn't_ always automatic, I suppose to not break legacy apps, so I'll update this